src/main/client/src/components/@commons/Warning.tsx

```diff
@@ -9,13 +9,17 @@ import { getToastTitle } from '../../utils/toastTitle'
 export const Warning: React.FC = () => {
   const [warning, setWarning] = React.useState<WarningView | undefined>(undefined)
   const [closed, setClosed] = React.useState<boolean>(false)
-
   const closeWarning = () => setClosed(true)
 
   React.useEffect(() => {
-    axios.get<WarningView>(`${API_BASE_URL}/api/warning`).then((res) => {
-      if (res.data.message !== '') setWarning(res.data)
-    })
+    axios
+      .get<WarningView>(`${API_BASE_URL}/api/warning`)
+      .then((res) => {
+        if (res.data.message !== '') setWarning(res.data)
+      })
+      .catch(() => {
+        console.error('Nem sikerült lekérni a figyelmeztetéseket.')
```

Comment:
Everywhere else `throwError()` is used, is there a reason why it's not used here?

Reply:
Because every navigation an error toast will be thrown. Tested, it is annoying as hell.